### DISCUSSION TITLE
Group action for selected rows in table

### DISCUSSION THREAD

**Discussion by joeDH00** [Budibase/budibase#13972] [Help]
Hi, I was wondering if i can run an action for group of data.

![image](https://github.com/Budibase/budibase/assets/159232925/ce8661cc-6935-418d-9a06-80203b58a184)

So i set up a table with row selection enable and have tried exporting data to csv so I wonder if it is possible to do it with different action. Added an action in the button as shown below but nothing happens.

![image](https://github.com/Budibase/budibase/assets/159232925/576abec9-2bdc-49fd-8765-bd9168bb471c)

version: 2.8.2
browser: chrome

**Comment by mikesealey** [comment]
Hey @joeDH00 

Add a button and set the On Click action to be "Export Data"
Select your table from the dropdown, along with the specifics of your preferred CSV 
![image](https://github.com/Budibase/budibase/assets/39269380/6c5278ff-87e0-4859-a852-c3796fbb548a)

Remember to add the columns you're looking for to your "Export Columns" action.
![image](https://github.com/Budibase/budibase/assets/39269380/3d67a5b8-7aa1-4e7b-acf7-c6068e73bcd3)

Then in the app, the user must check at least one row in the table, or the Export Data action will throw an error 'Please select at least one row".
![image](https://github.com/Budibase/budibase/assets/39269380/79bedf04-b2b8-4bfb-8358-338ff97772a0)

If the user selects rows and exports the data, it is downloaded as a `.csv` file, which can be viewed in most text editors. I prefer to use VSCode for it's colourscheme. 
![image](https://github.com/Budibase/budibase/assets/39269380/fffb7b68-401d-4597-afd8-05e2cb4fd9a2)

Don't forget you can also apply filters to the table based on bindings, which will offer even you more control over this feature.

**Reply by joeDH00** [reply]
i manage to "Export Data'. I was wondering if I can perform other actions besides "Export Data"
